frontend/app/routes/public/apply/$id/terms-and-conditions.tsx

```diff
@@ -33,6 +34,19 @@ export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
 export async function loader({ context: { appContainer, session }, request, params }: LoaderFunctionArgs) {
   loadApplyState({ params, session });
 
+  const applicationYears = await appContainer.get(TYPES.domain.services.ApplicationYearService).listApplicationYears({ date: '2020-01-01', userId: 'test' });
```

Comment:
We should change `date` to the current date and `userId`, by convention, to `anonymous` for our public routes (or add a `TODO` indicating that this should change)